### DISCUSSION TITLE
ColorPicker loses value in Wizard

### DISCUSSION THREAD

**Discussion by madsem** [filamentphp/filament#3264] [Help]
I have 2 ColorPicker fields in the first step of a Wizard.
When I manually change the value from the default, and continue, then go back, the values are gone.

I realized it because it is a required field, and on the final step when trying to submit nothing happened. No error was showing, so I went back and checked every step before.

So whenever I change anything in these two fields, I can't save the form anymore. At all, because whenever I change the values (Through picker or enter a hex code) it just deletes itself again and when trying to submit the form, I get an console error: `An invalid form control with name='' is not focusable.`

But when I don't touch these two fields, it all works perfectly.

```php
Card::make()
                ->schema([
                    Grid::make()
                        ->schema([
                            ColorPicker::make('header_color')
                                ->label('Header Color')
                                ->default(fn(string $state): string => $state ?? '#1f2937')
                                ->required()
                                ->hex(),

                            ColorPicker::make('footer_color')
                                ->label('Footer Color')
                                ->default(fn(string $state): string => $state ?? '#1f2937')
                                ->required()
                                ->hex(),
                        ]),
                ]),
```

I also tried removing the default etc, nothing helps.

**Comment by danharrin** [comment]
Possibly we are missing a wire:ignore on that input or something?
